riff-raff.yaml

```diff
@@ -0,0 +1,81 @@
+allowedStages:
+  - PROD
+deployments:
+  lambda-upload-eu-west-1-playground-cdk-playground-lambda:
+    type: aws-lambda
+    stacks:
+      - playground
+    regions:
+      - eu-west-1
+    app: cdk-playground-lambda
+    contentDirectory: cdk-playground-lambda
+    parameters:
+      bucketSsmLookup: true
+      lookupByTags: true
+      fileName: cdk-playground-lambda.zip
+    actions:
+      - uploadLambda
+  asg-upload-eu-west-1-playground-cdk-playground:
+    type: autoscaling
+    actions:
+      - uploadArtifacts
+    regions:
+      - eu-west-1
+    stacks:
+      - playground
+    app: cdk-playground
+    parameters:
+      bucketSsmLookup: true
+      prefixApp: true
+    contentDirectory: cdk-playground
+  cfn-eu-west-1-playground-cdk-playground:
+    type: cloud-formation
+    regions:
+      - eu-west-1
+    stacks:
+      - playground
+    app: cdk-playground
+    contentDirectory: cdk.out
+    parameters:
+      templateStagePaths:
+        PROD: CdkPlayground.template.json
+      amiParametersToTags:
+        AMICdkplayground:
+          BuiltBy: amigo
+          Recipe: arm64-bionic-java11-deploy-infrastructure
+          AmigoStage: PROD
+          Encrypted: true
```

Comment:
For ease, this file has been copied from the auto-generated version, with this `Encrypted` property the only addition.